projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/abilities/general/unbreakable.json

```
@@ -0,0 +1,14 @@
+{
+  "modifier_id": "tconstruct:unbreakable",
+  "text": [
+    {
+      "text": "毁坏的事物是无趣的，但谢天谢地有不毁的事物存在。"
+    }
+  ],
+  "effects": [
+    "工具不毁",
```

```suggestion
    "你的工具现在不毁了",
```
原文：Your tool is now unbreakable
这里建议尊重一下原文。（我觉得保持翻译风格很重要，前面风趣就一直风趣下去）